.github/actions/nm-install-test-whl/action.yml

```diff
@@ -64,6 +64,7 @@ runs:
         PYTHON_MAJOR_MINOR=$(echo "python${{ inputs.python }}" | cut -d'.' -f1,2)
         VLLM_SRC=$(python -c "import vllm; print(vllm.__path__[0])")
         ./.github/scripts/run-tests -s ${VLLM_SRC} -t ${{ inputs.test_directory }} -r ${{ inputs.test_results }} -f ${{ inputs.test_skip_list }}|| SUCCESS=$?
+        ./.github/scripts/run-tests -s ${VLLM_SRC} -t neuralmagic/tests -r ${{ inputs.test_results }} -f ${{ inputs.test_skip_list }}|| SUCCESS=$?
```

Comment:
This will overwrite the `SUCCESS` from the previous command, and thus not necessarily be the correct value. Instead, each of these commands would now need to add `$?` to `SUCCESS`.